src/BinSkim.Rules/PERules/BA2004.EnableSecureSourceCodeHashing.cs

```diff
@@ -164,6 +165,12 @@ public void AnalyzeNativeBinaryAndPdb(BinaryAnalyzerContext context)
                             {
                                 continue;
                             }
+                            else if (sf.FileName.EndsWith(".winmd"))
```

Comment:
>else [](http://example.com/codeflow?start=28&length=4)

Win RT components have references to metadata only files, which are used to link to OS provided API. These aren't source files, neither are they candidates to retrieve/render/debug in the IDE.